### DISCUSSION TITLE
[Question] Is there anyway to auto start the config/auto apply the rules?

### DISCUSSION THREAD

**Discussion by benyaminl** [sezanzeb/input-remapper#824] [Q&A] [answered]
Hello, I just want to ask, is there anyway to tell the input-remapper service, that we will auto apply the rule on start/restart service? Need some help regarding this. 

Thank you

**Comment by rankynbass** [comment, marked as answer]
Make sure you enable the service as described in the install instructions:  `sudo systemctl enable --now input-remapper`

It should auto-load your profiles on next reboot.
